Sources/Sentry/SentryCrashIntegration.m

```diff
@@ -244,27 +249,20 @@ - (void)configureScope
 
             [outerScope setContextValue:appData forKey:@"app"];
 
-            [outerScope addScopeListener:^(SentryScope *_Nonnull scope) {
-                // The serialization of the scope and synching it to SentryCrash can use quite some
-                // CPU time. We want to make sure that this doesn't happen on the main thread. We
-                // accept the tradeoff that in case of a crash the scope might not be 100% up to
-                // date over blocking the main thread.
-                [self.dispatchQueueWrapper dispatchAsyncWithBlock:^{
-                    NSMutableDictionary<NSString *, id> *userInfo =
-                        [[NSMutableDictionary alloc] initWithDictionary:[scope serialize]];
-
-                    // SentryCrashReportConverter.convertReportToEvent needs the release name and
-                    // the dist of the SentryOptions in the UserInfo. When SentryCrash records a
-                    // crash it writes the UserInfo into SentryCrashField_User of the report.
-                    // SentryCrashReportConverter.initWithReport loads the contents of
-                    // SentryCrashField_User into self.userContext and convertReportToEvent can map
-                    // the release name and dist to the SentryEvent. Fixes GH-581
-                    userInfo[@"release"] = self.options.releaseName;
-                    userInfo[@"dist"] = self.options.dist;
-
-                    [SentryCrash.sharedInstance setUserInfo:userInfo];
-                }];
-            }];
+            NSMutableDictionary<NSString *, id> *userInfo =
+                [[NSMutableDictionary alloc] initWithDictionary:[outerScope serialize]];
+            // SentryCrashReportConverter.convertReportToEvent needs the release name and
+            // the dist of the SentryOptions in the UserInfo. When SentryCrash records a
+            // crash it writes the UserInfo into SentryCrashField_User of the report.
+            // SentryCrashReportConverter.initWithReport loads the contents of
+            // SentryCrashField_User into self.userContext and convertReportToEvent can map
+            // the release name and dist to the SentryEvent. Fixes GH-581
```

Comment:
```suggestion
            // the release name and dist to the SentryEvent.
```

Reply:
Don't you think it's a good idea to keep a reference to the related issue?

Reply:
if its an issue not connected to the PR description or linked issue, yes, but if its related to the very same PR/issue, I'd say not, I didn't check if its a totally separated issue or not

Reply:
FYI, this was already there. I jut moved the code around.